# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../espeak/install.patch
# Copyright (C) 2017 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- espeak-1.48.04-source/src/Makefile.vanilla	2017-09-19 00:40:48.455871697 +0200
+++ espeak-1.48.04-source/src/Makefile	2017-09-19 00:40:29.619872526 +0200
@@ -147,7 +147,7 @@
 	$(LN_SF) $(LIB_NAME).$(LIBTAG) $(DESTDIR)$(LIBDIR)/$(LIB_NAME).$(LIB_VERSION)
 	$(LN_SF) $(LIB_NAME).$(LIB_VERSION) $(DESTDIR)$(LIBDIR)/$(LIB_NAME)
 	# Install development headers
-	$(INSTALL) -pm 644 speak_lib.h $(DESTDIR)$(INCDIR)
+	$(INSTALL) -p -m 644 speak_lib.h $(DESTDIR)$(INCDIR)
 	# Install data files
 	cp -prf ../espeak-data/* $(DESTDIR)$(DATADIR)
 
